src/coldstorage-subaccount/ColdStorageHook.sol

```diff
@@ -25,7 +25,13 @@ contract ColdStorageHook is ERC7579HookDestruct {
     mapping(address subAccount => VaultConfig) internal vaultConfig;
     mapping(address subAccount => EnumerableMap.Bytes32ToBytes32Map) internal executions;
 
-    event WithdrawalRequested(address indexed subAccount, IERC7579Execution.Execution indexed exec);
+    event ExecutionRequested(
+        address indexed subAccount,
+        IERC7579Execution.Execution indexed exec,
```

Comment:
@kopy-kat are  `exec` and `executeAfter` need to be indexed?

Reply:
yeah good shout, for the subgraph I don't think they need to be indexed

Reply:
it would only help with querying logs directly but we dont need to do that